docs/how-to-guides/deploy-a-flow/flow-model-packaging.md

```diff
@@ -0,0 +1,204 @@
+# Flow model packaging
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+There are Four steps to package a flow and deploy in locals:
+1. Build the flow as docker format.
+2. Prepare an entry file.
```

Comment:
Looks like we missed a build --format executable source code change?

Reply:
updated